AcculturationGuide/acculturation.rst

```diff
@@ -256,8 +264,8 @@ Note that you may have to install the program 'tree' before running the above co
 With YottaDB, you can write applications that implement international character sets using Unicode or ISO/IEC-10646 (the two standards track each other). Connect to the virtual machine with your terminal emulator configured to support the UTF-8 character set. In a fresh terminal session execute the following (the non-printable characters may look different on your session from the screen here, depending on how your terminal emulator renders them):
 
 .. parsed-literal::
-   yottadbuser@yottadbworkshop:~$ export gtm_chset=UTF-8 LC_CTYPE=en_US.utf8
-   yottadbuser@yottadbworkshop:~$ source /usr/local/lib/yottadb/r120/ydb_env_set
+   yottadbuser@yottadbworkshop:~$ export ydb_chset=UTF-8 LC_CTYPE=en_US.utf8
```

Comment:
Does YottaDB accept ydb_chset yet?

Reply:
Not yet.

Reply:
I thought ydb_env_set set it. I can revert it back to gtm_chset.

Reply:
To clarify my response, libyottadb.so/mumps/mupip/dse etc. have no idea of "ydb_chset" as of now (even though ydb_env_set might set this env var). They still look at "gtm_chset".